var/spack/repos/builtin/packages/qhull/package.py

```diff
@@ -47,8 +48,7 @@ class Qhull(Package):
     
     depends_on('cmake')
 
-    def install(self, spec, prefix):
-        with working_dir('spack-build', create=True):
-            cmake('..', *std_cmake_args)
-            make()
-            make("install")
+    @CMakePackage.sanity_check('build')
+    @CMakePackage.on_package_attributes(run_tests=True)
```

Comment:
CMake's main claim to fame is it's not as arcane as Autotools.  Here's to wishing there were a Python-based CMake-like system that was as easy to use as Spack.